src/material-experimental/mdc-slider/slider-thumb.ts

```diff
@@ -33,42 +60,43 @@ export interface MatSliderDragEvent {
  * The native input used by the MatSlider.
  */
 @Directive({
-  selector: 'input[mat-slider-thumb]',
+  selector: 'input[matSliderThumb], input[matSliderStartThumb], input[matSliderEndThumb]',
   host: {
     'class': 'mdc-slider__input',
     'type': 'range',
-    '[min]': 'min',
-    '[max]': 'max',
-    '[step]': 'step',
-    '[attr.value]': 'value',
     '(blur)': '_blur.emit()',
     '(focus)': '_focus.emit()',
-  }
-}) export class MatSliderThumb {
-  /** The current value of this slider input. */
-  @Input()
-  get value(): number { return this._value; }
-  set value(v: number) { this._value = v; }
-  private _value: number;
+  },
+}) export class MatSliderThumb implements AfterViewInit {
+
+  // ** IMPORTANT NOTE **
+  //
+  // The way `value` is implemented for MatSliderThumb goes against our standard practice. Normally
+  // we would define a private variable `_value` as the source of truth for the value of the slider
+  // thumb input. The source of truth for the value of the slider inputs has already been decided
```

Comment:
Couldn't we work around this by checking in the adapter's `{set,get}Attribute` method if the attribute name being set is `value`? If it is we can just update/read whatever we consider to be the real source of truth, rather than the attribute.

Reply:
I suggested this to Jeremy a few days ago and he said to leave it as is and just add a comment explaining why we do it this way

Reply:
Yeah, rather than fight against the design decision that MDC made that the value attribute is the source of truth (which is a very wiz mentality), I figured it's less work to just leave it.